schema/geometry_schema.json

```diff
@@ -1,5 +1,6 @@
 {
 	"$schema": "http://json-schema.org/draft-04/schema#",
+	"version": "0.3.0",
```

Comment:
@macumber is this acceptable in the schema? I've never used json-schema before this.